roles/docker/tasks/main.yml

```diff
@@ -1,10 +1,88 @@
 ---
 # tasks file for docker
+
+- name: Set node facts applicable to docker
+  openshift_facts:
```

Comment:
I'd like to avoid adding a requirement on openshift_facts to the docker role, since it is meant to be a generic docker role and not OpenShift specific.

Instead, I would prefer to see the OpenShift specific config managed through the playbooks or through an openshift_docker role.

That said, I do think that the log options and registry settings should belong in the docker role, just not the references to openshift_facts varaibles, openshift_* variables, or any of the filters defined in oo_filters.